packages/design-system/src/types/UsaBanner/UsaBanner.d.ts

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+
+export type LocaleLanguage = 'en' | 'es';
+
+export interface UsaBannerProps {
+  /**
+   * Additional classes to be added to the root `section` element
+   */
+  className?: string;
+  /**
+   * A unique ID to be applied to the banner content. A unique ID will be generated if one isn't provided.
+   */
+  id?: string;
+  /**
+   * The language the USA Banner will render as.
```

Comment:
Update the text here to read a bit more friendly 

```suggestion
   * The language the USA Banner will be presented in.
```